cob_bringup/tools/docker_control.launch

```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
```

Comment:
I had the same question but my best guess is that the node may be obsolete or has been updated to ``multi_station_docker_control.py`` and was not removed from here.